cglims/api.py

```diff
@@ -11,6 +11,49 @@ def connect(config):
     api = ClinicalLims(config['host'], config['username'], config['password'])
     return api
 
+class ClinicalSample:
+
+    def __init__(self, sample):
+        """ Wrapper around the genologics Sample class
+
+        Args:
+            sample (genologics.Sample): the sample instance to extend
+        """
+        self.lims = sample
+        self._pipeline = None
```

Comment:
I think the calculation of the "pipeline" property is simple enough the caching maybe is adding more complexity than needed, what do you think?